crates/bevy_transform/src/components/transform.rs

```diff
@@ -63,43 +108,48 @@ impl Transform {
         }
     }
 
-    /// Returns transform with the same translation and scale, but rotation so that
-    /// transform.forward() points at target
+    /// Update and return this [`Transform`] by rotating it so that its unit vector in the
+    /// local x direction is toward `target` and its unit vector in the local y direction
```

Comment:
"unit vector in the local x direction is toward target" should be "local z direction"